velox/functions/lib/Re2Functions.cpp

```diff
@@ -439,6 +445,8 @@ class OptimizedLikeWithMemcmp final : public VectorFunction {
         return matchPrefixPattern(input, pattern, reducedPatternLength);
       case PatternKind::kSuffix:
         return matchSuffixPattern(input, pattern, reducedPatternLength);
+      case PatternKind::kFixedMiddle:
+        return matchOptimizedPattern(input, pattern);
```

Comment:
Change name of function to matchFixedMiddle as well.

Reply:
Thanks for pointing it out, updated.